runtime/lib/backend/lightning/lightning_dynamic/LightningSimulator.cpp

```diff
@@ -108,14 +108,16 @@ void LightningSimulator::SetState(DataView<std::complex<double>, 1> &data,
                                   std::vector<QubitIdType> &wires)
 {
     std::vector<std::complex<double>> data_vector(data.begin(), data.end());
-    std::vector<std::size_t> wires_size_t(wires.begin(), wires.end());
+    auto &&dev_wires = getDeviceWires(wires);
+    std::vector<std::size_t> wires_size_t(dev_wires.begin(), dev_wires.end());
```

Comment:
I don't think we need to repackage the wires into an additional vector

Reply:
This was suggested on the lightning side too (https://github.com/PennyLaneAI/pennylane-lightning/pull/869), I will change this